backend-models/service-tests/courseedservice-tests.txt

```diff
@@ -0,0 +1,258 @@
+
+courseedservice.php
+
+---------------------------------------------------------------------------------------------------------------------
+- Insert into course (line 79)
+---------------------------------------------------------------------------------------------------------------------
+
+*********
+* pre-req:
+********* 
+
+{(checklogin) && ($ha)} IS TRUE
+
+*********
+* send parameters inputs to courseedservice.php 
+*********
+send
+{
+    $opt = "NEW";
+    $userid = "UNK"; or $userid = !NULL;
+    $coursecode = "G1337";
+    $coursename = "Testing-Course";
+    $courseGitURL = NULL; // There are no values in the database
```

Comment:
i had problems doing inserts into the table with the INSERT string found in the service file. 
might have to do with the fact that the names of the colums are different. 
![image](https://user-images.githubusercontent.com/102600391/235890227-6b289255-cdc3-47d8-bcb5-a2d5de69a5d2.png)
![image](https://user-images.githubusercontent.com/102600391/235890311-093335ed-0d3d-4f48-9116-8feeeedbdbf9.png)

there is of coarse nothing you can do about it (or rather should not do), but it can be a good idea to document it.